SU2_CFD/include/numerics/NEMO/convection/ausm.hpp

```diff
@@ -37,13 +37,25 @@
  */
 class CUpwAUSM_NEMO : public CNEMONumerics {
 private:
-  su2double *FcL, *FcR;
-  //su2double *FcLR;
-  //su2double *dmLP, *dmRM, *dpLP, *dpRM;
-  //su2double *daL, *daR;
+  su2double *FcL, *FcR, *FcLR;
+  su2double *dmLP, *dmRM, *dpLP, *dpRM;
+  su2double *daL, *daR;
   su2double ProjVel_i, ProjVel_j;
-  su2double sq_vel;
-  //su2double  Proj_ModJac_Tensor_ij;
+  su2double sq_vel, Proj_ModJac_Tensor_ij;
+
+  /************************************/
+  //DELETE ME TODO: ROE JACOBIAN STUFF
+  /*__________________________________*/
+  su2double *Diff_U;
```

Comment:
These are only here at the moment for testing the AUSM scheme with ROE Jacobian in comparison with the flow solver.